### PR TITLE
Splunk ext

### PR DESCRIPTION
**Pull request by AmitHaim1**
add mapping to x_oca_event, add some tests respectively

### CODE REVIEW

File: stix_shifter_modules/splunk/stix_translation/json/select_fields.json [hunk 1]
```diff
@@ -1,28 +1,53 @@
...
+    "parent_process_exec",
+    "description",
+    "result",
+    "signature",
+    "signature_id",
+    "query",
+    "answer"
+  ]
 }
+
```
Comment (mdazam1942) [inline stix_shifter_modules/splunk/stix_translation/json/select_fields.json:45]: please remove these extra lines

**Review by mdazam1942 — COMMENTED** [review]
there are few new fields added in the mapping. are these fields available by default in the API? Can you confirm if there is no extension or app needed to install in splunk in order to make these fields available?

File: stix_shifter_modules/splunk/stix_translation/transformers.py [hunk 2]
```diff
@@ -6,4 +6,44 @@ class SplunkToTimestamp(ValueTransformer):
 
     @staticmethod
     def transform(splunkTime):
-        return splunkTime[:-6]+'Z'
+        return splunkTime[:-6] + 'Z'
+
+
+class SplunkHash(ValueTransformer):
```
[the remark this answers is not in the document]
Comment (AmitHaim1) [reply stix_shifter_modules/splunk/stix_translation/transformers.py:12]: Added check of this transformer under test_process_cim_to_stix() on test_splunk_json_to_stix.py

**Comment by AmitHaim1** [comment]
> These fields are available by default in the API, you can check on the Splunk doc - https://docs.splunk.com/Documentation/CIM/5.0.0/User/CIMfields

**Review by mdazam1942 — APPROVED** [review]

**Pull request merged**